playwright/mockData/orgs/KPD/journeys/instances/ClarasOnboarding.ts

```diff
@@ -0,0 +1,46 @@
+import ActivistTag from '../../tags/Activist';
+import ClaraZetkin from '../../people/ClaraZetkin';
+import MemberOnboarding from '../MemberOnboarding';
+import PlaysGuitarTag from '../../tags/PlaysGuitar';
+import { ZetkinJourneyInstance } from 'types/zetkin';
+
+const ClarasOnboarding: ZetkinJourneyInstance = {
+  assigned_to: [ClaraZetkin],
+  created_at: '2022-02-01T14:53:15',
+  id: 1,
+  journey: {
+    id: MemberOnboarding.id,
+    title: MemberOnboarding.singular_name,
+  },
+  milestones: [
+    {
+      deadline: '2022-03-06T14:53:15',
+      status: 'completed',
+      title: 'Sign up for newsletter',
+    },
+    {
+      deadline: '2022-05-23T12:53:15',
+      status: 'unfinished',
+      title: 'Go to open meeting',
+    },
+    {
+      deadline: '2022-07-11T12:00:00',
+      status: 'unfinished',
+      title: 'Fill out member survey',
+    },
+  ],
+  next_milestone: {
+    deadline: '2022-05-23T12:53:15',
+    status: 'unfinished',
+    title: 'Go to open meeting',
+  },
+  people: [ClaraZetkin],
+  status: 'open',
```

Comment:
This could also be an enum right? So there's a journey instance status and a milestone status.